packages/composites/lido/test-payload.json

```diff
@@ -0,0 +1,3 @@
+{
+ "requests": [{}]
```

Comment:
This should contain something to be used for smoke tests

Reply:
There is no required input params, so I just added an "id"

Reply:
Ah yeah I see that now.